.eslintrc.js

```diff
@@ -127,6 +127,9 @@ module.exports = {
     // https://eslint.org/docs/latest/rules/no-undef#handled_by_typescript
     "no-undef": "error",
 
+    "@typescript-eslint/array-type": ["error", {
+      default: "array"
```

Comment:
oh nice, thank you for adding this explicitly! 

I realized that I have been inconsistent about this, but I think it makes sense to have the option block provided explicitly, even if it matches the default, when it's not obvious from the rule name what our "choice" is.